packages/chat-component-bindings/src/utils/getParticipantsWhoHaveReadMessage.test.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { ChatMessageWithStatus } from '@internal/chat-stateful-client';
+import { getParticipantsWhoHaveReadMessage } from './getParticipantsWhoHaveReadMessage';
+
+const chatMessage = (messageID: string): ChatMessageWithStatus => ({
+  id: messageID,
+  type: 'text',
+  sequenceId: '89542038450',
+  version: 'jehkfhsdf',
+  createdOn: new Date(),
+  status: 'delivered'
+});
+
+const readReceiptForEachSender = {
+  '12345': { lastReadMessage: '3', name: 'c' },
+  '67890': { lastReadMessage: '5', name: 'd' },
+  '11111': { lastReadMessage: '3', name: 'a' },
+  '22222': { lastReadMessage: '1', name: 'l' },
```

Comment:
we can give these names a more meaningful word, for better understanding later

Like 
  '11111': { lastReadMessage: '3', name: 'UserWhoReadsTillMessage3' },
  '22222': { lastReadMessage: '1', name: 'UserWhoReadsTillMessage1' },
  
 Then all the test result below would be more readable